src/init.ts

```diff
@@ -1,23 +1,25 @@
-import * as monaco from 'monaco-editor'
+// @ts-nocheck
+import * as json from 'monaco-editor/esm/vs/language/json/monaco.contribution'
```

Comment:
I ignore the type checking because `monaco-editor` did not expose correct typing file for `monaco-editor/esm/vs/language/json/monaco.contribution`

Reply:
![image](https://github.com/user-attachments/assets/83076061-3403-405f-b718-5cdeb47b6f99)